lib/sgtn-client/util/locale-util.rb

```diff
@@ -10,6 +10,11 @@ module SgtnClient
                   }
 
     class LocaleUtil
+        def self.process_locale(locale=nil)
+            locale = locale.class == Hash ? locale[:locale] : locale
```

Comment:
In what case `locale` maybe a Hash?

Reply:
same question. how the caller know it's used as locale[:locale], why not locale[:language] and etc.?

Reply:
Is this not a valid call: `SgtnClient::Translation.getString("JAVA", "helloworld", locale:nil)`? If not, then I could remove hash handling. Even simpler.

Reply:
I have removed the hash case.